data/json/mapgen/lab/lab_floorplans_finale1level.json

```
@@ -177,7 +177,8 @@
             { "item": "sample_t_substrate" },
             { "item": "log_immersion27" },
             { "item": "HAADF_t_substrate" },
-            { "item": "report_t_substrate" }
+            { "item": "report_t_substrate" },
+            { "item": "id_science"}
```

```suggestion
            { "item": "id_science" }
```

Needs a whitespace after the double quotes for the JSON Validation.

---

oops, thanks for the catch! probably should get local verification working at some point